xwiki-commons-core/xwiki-commons-velocity/src/test/java/org/xwiki/velocity/tools/URLToolTest.java

```diff
@@ -81,4 +84,18 @@ void preserveParameterOrder()
         String queryString = "x=5&x=4&r=3&a=2";
         assertEquals(queryString, escapeTool.url(tool.parseQuery(queryString)));
     }
+
+    @Test
+    void getURLValid() throws MalformedURLException
```

Comment:
As suggested by @vmassol in a previous comment, the name of the test method should use the name of the tested method.